BritneyS/GetFed
# Bugfix no data

## What you did :question:
- Removed all macronutrient labels on the Food Detail view if any macronutrient data was not returned for and Food entry

## How you did it :white_check_mark:
- Wrote method `checkMacronutrientData` to check if protein, carbs, or fat values return nil, and hide labels if any of them do return nil


## How to test it :microscope:
- Run the app
- Tap 'Food Search'
- Search for 'Apple'
- Tap the cell that says 'Apple' and 'Apples'
- Note that the message 'No chart data is available' is displayed, no macronutrient values are displayed, and the calories value is still visible (see screenshot labeled **AFTER FIX**)


## Screenshots (if applicable) :camera:
**BEFORE FIX**
![simulator screen shot - iphone xr - 2018-12-06 at 13 37 01](https://user-images.githubusercontent.com/8409475/50182694-0b793980-02de-11e9-89d3-a5ae1282fdb2.png)

**AFTER FIX**
![simulator screen shot - iphone xr - 2018-12-18 at 16 03 01](https://user-images.githubusercontent.com/8409475/50182937-aa9e3100-02de-11e9-9666-dc6e83c5ef04.png)
